application/views.py

```diff
@@ -13,12 +14,25 @@ def dashboard():
     context["title"] = "Dashboard"
     if request.method == 'POST':
         payload = request.get_json()
-        response = jsonify(dict(
-            status="success",
-            message="successfully saved request(s)"
-        ))
-        response.status_code = 200
-        return response
+        
+        def handle_request(n):
+            if n < 1:
+                return "no data"
+            else:
+                n -= 1
+                req = Request(
+                    collection_id=2,
```

Comment:
Why is this hard coded?

Reply:
because by the time of the PR, the payload sent doesn't contain the collection id